docs/mkdocs/docs/technical/releasing.md

```diff
@@ -0,0 +1,57 @@
+# Release Tooling
+
+This document details the release process for ArcticDB. ArcticDB is released onto [PyPi](https://pypi.org/project/arcticdb/) and [Conda](https://anaconda.org/conda-forge/arcticdb).
```

Comment:
Please break long lines in markdown

Reply:
All long lines split. Thanks for the reminder.